pkg/bootstrap/bootstrap_config.go

```diff
@@ -239,16 +239,19 @@ type istioMetadata struct {
 	Labels                    map[string]string `json:"labels,omitempty"`
 	Name                      string            `json:"name,omitempty"`
 	Namespace                 string            `json:"namespace,omitempty"`
+	Owner                     string            `json:"owner,omitempty"`
+	PortsToContainers         map[string]string `json:"ports_to_containers,omitempty"`
 	ServiceAccount            string            `json:"service_account,omitempty"`
+	WorkloadName              string            `json:"workload_name,omitempty"`
 	PlatformMetadata          map[string]string `json:"platform_metadata,omitempty"`
 }
 
 func shouldExtract(envVar, prefix string) bool {
-	// this will allow transition from current method of exposition in the future
-	// Example:
-	// if strings.HasPrefix(envVar, "ISTIO_METAJSON_LABELS") {
-	// 	return false
-	// }
+	if strings.HasPrefix(envVar, "ISTIO_META_OWNER") ||
```

Comment:
Why does this need special treatment ?

Reply:
We don't want to duplicate this information directly into the other metadata sections. without this special treatment, you'd see these values twice, once at a root level and once in the intended location "istio.io/metadata".  This preserves existing behavior of putting all the labels directly in the root metadata level for backwards compatibility.